_tuning-your-cluster/availability-and-recovery/workload-management.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Workload management
+nav_order: 60
+has_children: false
+parent: Availability and recovery
+---
+
+# Workload management
+
+Workload management is a mechanism that allows administrators to organize queries into distinct groups, referred to as _query groups_. These query groups enable admins to limit the cumulative resource usage of each group, ensuring more balanced and fair resource distribution between them. This mechanism provides greater control over resource consumption, so that no single group can monopolize cluster resources at the expense of others.
```

Comment:
```suggestion
Workload management is a mechanism that allows administrators to organize queries into distinct groups, referred to as _query groups_. These query groups enable admins to limit the cumulative resource usage of each group, ensuring more balanced and fair resource distribution between them. This mechanism provides greater control over resource consumption so that no single group can monopolize cluster resources at the expense of others.
```